ecommerce/extensions/basket/models.py

```diff
@@ -130,6 +130,8 @@ class Meta(object):
 
 class BasketChallanVoucher(TimeStampedModel):
```

Comment:
Can we add docstring for the purpose of this class?
plus with methods where needed?

Reply:
sure